src/commands/dora/README.md

```diff
@@ -0,0 +1,85 @@
+# DORA Metrics (beta)
```

Comment:
To make the command beta, you have to add it to `BETA_COMMANDS` here:

https://github.com/DataDog/datadog-ci/blob/78f71731879edb0913ba99c16f695a53acf96953/src/cli.ts#L8

And it will have to be enabled with `DD_BETA_COMMANDS_ENABLED=1` (no need to put this in the README).

Reply:
Interesting I didn't know about this thanks 👍